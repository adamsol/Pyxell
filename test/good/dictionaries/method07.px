
d = {"x": 1, "yz": 5}
print d.has('x')
print d.has("yz")
print d.has("xyz")

d' = {for i in -3...-10^^4 by -1 yield i: false}
for i in -10^^4...10^^4 do
    if d'.has(i) != (i < 0) do
        print i
