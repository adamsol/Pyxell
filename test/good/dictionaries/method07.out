true
true
false
-10000
-2
-1
